ext/handlers/filters/updates.go

```diff
@@ -0,0 +1,61 @@
+package filters
+
+import (
+	"github.com/PaulSonOfLars/gotgbot/v2"
+)
+
+type ChatMember func(u *gotgbot.ChatMemberUpdated) bool
+
+func (f ChatMember) And(f2 ChatMember) ChatMember {
+	return func(cq *gotgbot.ChatMemberUpdated) bool {
+		return f(cq) && f2(cq)
+	}
+}
+
+func (f ChatMember) Or(f2 ChatMember) ChatMember {
+	return func(cq *gotgbot.ChatMemberUpdated) bool {
+		return f(cq) || f2(cq)
+	}
+}
+
+func (f ChatMember) Not() ChatMember {
+	return func(cq *gotgbot.ChatMemberUpdated) bool {
+		return !f(cq)
+	}
+}
+
+func ChatMemberUserId(id int64) ChatMember {
+	return func(cm *gotgbot.ChatMemberUpdated) bool {
+		return cm.NewChatMember.User.Id == id
+	}
+}
+
+func PerformerUserId(id int64) ChatMember {
```

Comment:
"performer" is a new term that hasn't been used in other contexts. What do you think of changing this to `ChatMemberFromUserId`?

Reply:
Ok

Reply:
Done.